tensorflow/contrib/framework/python/framework/graph_util.py

```diff
@@ -126,3 +128,27 @@ def fuse_op(graph_def, input_nodes, output_nodes, output_dtypes,
   out.library.CopyFrom(graph_def.library)
   out.versions.CopyFrom(graph_def.versions)
   return out
+
+
+def get_placeholders(graph):
+  """Get placeholders of a graph.
+
+  Args:
+    graph: A tf.Graph.
+  Returns:
+    A list contains all placeholders of given graph.
+
+  Raises:
+    TypeError: if 'graph' is not a tensorflow graph.
```

Comment:
Sorry, my mistake. I mean that graph should be surrounded by back-ticks `, instead of single quote '. 

TypeError: If ` graph`